modules/ROOT/pages/introduction/intro-config-use-fd.adoc

```diff
@@ -0,0 +1,142 @@
+= Use Flow Designer to Configure a Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Flow Designer is a tool in Anypoint Design Center that enables you to create apps visually. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To use Flow Designer to configure a connector:
+
+. Configure the input source (trigger) for the flow.
+. Configure the target connector operation.
+
+[[configure-input-source]]
+== Configure the Input Source for the Flow
+
+. Log into Anypoint Platform.
+. Click *Start designing*.
+. In Design Center, click *Create new*.
+. Click *Create new application*.
+. Enter a project name and click *Create*.
+. Click *Go straight to canvas* to exit from *Let's get started*.
++
+.Clicking *Go straight to canvas* takes you to a trigger card.
+image::introduction/intro-config-fd-go-to-canvas.png[Exit from the Let's Get Started screen]
++
+. Click on the empty trigger card in the middle of the canvas to expand it.
+. Configure one of these input sources for the connector:
+* <<http-listener,HTTP Listener>>
+* <<scheduler,Scheduler>>
+* <<connector-specific,A connector-specific input source>>
+
+[[http-listener]]
+=== Use HTTP Listener as an Input Source
+
+`HTTP Listener` listens for HTTP requests on a specified path and initiates the associated flow when it receives a request. To use `HTTP Listener` as an input source:
+
+. Follow the procedure described in <<configure-input-source,Configure an Input Source for the Flow>>.
+. In the *Search* field, enter *HTTP*.
+. In the *Operation* field, click *HTTP Listener*.
+. In the *Path* field, specify the path on which `HTTP Listener` listens for requests.
+. Click *Edit*:
++
+.Clicking *Edit* takes you to a screen where you can configure the HTTP Listener connection.
+image::introduction/intro-config-fd-exit.png[CLick Edit]
++
+. Click *Add Connection*.
+. In the *Connection Name* field, enter a connection name.
+. Select *Listener* as the connection type.
+. Enter values for the *Host* and *Port* fields:
+* To listen on a specific port for every network interface on the server, leave the default value `0.0.0.0` for *Host*.
+* To listen for incoming requests generated inside the server, set the *Port* value to `localhost`.
```

Comment:
I tested this and I don't believe you can actually type the text "localhost" into port. You need to use the actual port number

Reply:
Ashley is correct. You can put localhost for host, but you have to use a port number in the Port field.